in_toto/util.py

```diff
@@ -124,7 +124,7 @@ def prompt_import_rsa_key_from_file(filepath):
   password = None
   try:
     import_rsa_key_from_file(filepath)
-  except securesystemslib.exceptions.CryptoError, e:
+  except securesystemslib.exceptions.CryptoError:
```

Comment:
I wonder, is the lack of password a `CryptoError` specifically. Isn't there another case that could cause this behavior and ask for a password to a key it didn't even load?